extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -647,7 +646,16 @@ export class AmpAnalytics extends AMP.BaseElement {
         return;
       }
       this.expandAndSendRequest_(request, trigger, event);
-      this.expandAndPostMessage_(trigger, event);
+
+      const shouldSendToAmpAd =
+        trigger['parentPostMessage'] &&
+        this.allowParentPostMessage_() &&
+        isIframed(this.win);
+      if (shouldSendToAmpAd) {
+        this.expandEventToMessage_(trigger, event).then((message) => {
+          this.win.parent./*OK*/ postMessage(message, '*');
+        });
+      }
```

Comment:
nit: since `expandEventToMessage_` is barely doing any work now, I would either move that logic here or move this logic to `expandEventToMessage_` and revert the name change.

Reply:
Done!